var/spack/repos/builtin/packages/gcc-runtime/package.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import re
+
+from spack.package import *
+from spack.util.elf import parse_elf
+
+
+class GccRuntime(Package):
+    """Package for GCC compiler runtime libraries"""
+
+    homepage = "https://gcc.gnu.org"
+    has_code = False
+
+    maintainers("haampie")
+
+    license("GPL-3.0-only")
+
+    requires("%gcc")
+
+    for v in [
+        "13.2",
+        "13.1",
+        "12.3",
+        "12.2",
+        "12.1",
+        "11.4",
+        "11.3",
+        "11.2",
+        "11.1",
+        "10.5",
+        "10.4",
+        "10.3",
+        "10.2",
+        "10.1",
+        "9.5",
+        "9.4",
+        "9.3",
+        "9.2",
+        "9.1",
+        "8.5",
+        "8.4",
+        "8.3",
+        "8.2",
+        "8.1",
+        "7.5",
+        "7.4",
+        "7.3",
+        "7.2",
+        "7.1",
+        "6.5",
+        "6.4",
+        "6.3",
+        "6.2",
+        "6.1",
+        "5.5",
+        "5.4",
+        "5.3",
+        "5.2",
+        "5.1",
+        "4.9.4",
+        "4.9.3",
+        "4.9.2",
+        "4.9.1",
+        "4.9.0",
+        "4.8.5",
+        "4.8.4",
+        "4.8.3",
+        "4.8.2",
+        "4.8.1",
+        "4.8.0",
+        "4.7.4",
+        "4.7.3",
+        "4.7.2",
+        "4.7.1",
+        "4.7.0",
+        "4.6.4",
+        "4.6.3",
+        "4.6.2",
+        "4.6.1",
+        "4.6.0",
+        "4.5.4",
+        "4.5.3",
+        "4.5.2",
+        "4.5.1",
+        "4.5.0",
+    ]:
+        version(v)
+        requires(f"%gcc@{v}", when=f"@{v}")
+
+    def install(self, spec, prefix):
```

Comment:
Stuff in this function is :100: and I believe we're going to use it, one way or another.